rules/linux/persistence_suspicious_ssh_execution_xzbackdoor.toml

```diff
@@ -0,0 +1,102 @@
+[metadata]
+creation_date = "2024/04/01"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2024/04/01"
+
+
+[rule]
+author = ["Elastic"]
+description = """
+It identifies potential malicious shell executions through remote SSH and detects cases where the sshd service suddenly
+terminates soon after successful execution, suggesting suspicious behavior similar to the XZ backdoor.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Potential Execution via XZBackdoor"
+references =["https://github.com/amlweems/xzbot", "https://access.redhat.com/security/cve/CVE-2024-3094"]
+risk_score = 73
+rule_id = "7afc6cc9-8800-4c7f-be6b-b688d2dea248"
+severity = "high"
+tags = [
+        "Domain: Endpoint",
+        "OS: Linux",
+        "Use Case: Threat Detection",
+        "Tactic: Credential Access",
+        "Tactic: Persistence",
+        "Tactic: Lateral Movement",
+        "Data Source: Elastic Defend"
+        ]
+timestamp_override = "event.ingested"
+type = "eql"
+query = '''
+sequence by host.id with maxspan=1s
+ [process where host.os.type == "linux" and event.action == "exec" and process.name == "sshd"] by process.pid, process.entity_id
```

Comment:
Could choose to add `event.type` before `event.action` for readability, but that's just a preference.

Reply:
done